bdf-parser/Cargo.toml

```
@@ -1,23 +1,13 @@
 [package]
 name = "bdf-parser"
-description = "Parse BDF files"
+description = "BDF font parser"
 version = "0.1.0"
 edition = "2018"
-authors = ["James Waples <jamwaffles@gmail.com>"]
-repository = "https://github.com/jamwaffles/embedded-fonts"
-documentation = "https://jamwaffles.github.io/embedded-fonts"
+authors = ["James Waples <jamwaffles@gmail.com>", "Ralf Fuest <mail@rfuest.de>"]
```

@jamwaffles Is this OK for you? I'm not sure if I can publish a crate when I'm not an author. I'll add ownership for the core GitHub team when I've published the crate.

---

That's fine, although my email changed a while back :). Might as well update it while we're here.

```suggestion
authors = ["James Waples <james@wapl.es>", "Ralf Fuest <mail@rfuest.de>"]
```